app/helpers/urls.py

```diff
@@ -43,7 +43,7 @@ def create_url(table, url):
         now = time.localtime()
         table.put_item(
             Item={
-                'url_short': shortened_url,
+                'shortlinks_id': shortened_url,
```

Comment:
Perhaps this "shortened_url" should be changed as well.

Reply:
would certainly make it clearer to use the same names consistently